src/utils/env.ts

```diff
@@ -13,3 +13,7 @@ export function isStagingEnv(): boolean {
 export function isProductionEnv(): boolean {
   return process.env.NODE_ENV === 'production' && !isStagingEnv()
 }
+
+export function isSentryEnabled(): boolean {
+  return process.env.REACT_APP_SENTRY_DISABLED !== 'true'
```

Comment:
Could you flip the boolean to `REACT_APP_SENTRY_ENABLED === 'true'`, so that it is only enabled if explicitly enabled? Otherwise someone using their own .env file may accidentally enable it by not explicitly disabling it.